backend/checkmach.ml

```diff
@@ -50,6 +50,8 @@ end
 module Func_info = struct
   type t =
     { name : string;  (** function name *)
+      mutable dbg : Debuginfo.t option;
```

Comment:
If we are only interested in the
location, it feels like the value
stored here is rather large. We
could apply the
`Debuginfo.to_location (Option.get ...)`
early to store a smaller value.

Reply:
Fixed.